test/e2e/antreapolicy_test.go

```diff
@@ -930,6 +932,40 @@ func testCNPRulePrioirty(t *testing.T) {
 	executeTests(t, testCase)
 }
 
+// testCNPPortRange tests the port range in a CNP can work.
+func testCNPPortRange(t *testing.T) {
```

Comment:
```suggestion
func testACNPPortRange(t *testing.T) {
```

Reply:
preferably use acnp/ACNP instead of cnp/CNP

Reply:
Sure.